java/client/src/org/openqa/selenium/remote/http/JsonHttpCommandCodec.java

```diff
@@ -133,6 +134,8 @@ public JsonHttpCommandCodec() {
     defineCommand(TOUCH_MOVE, post("/session/:sessionId/touch/move"));
     defineCommand(TOUCH_SCROLL, post("/session/:sessionId/touch/scroll"));
     defineCommand(TOUCH_UP, post("/session/:sessionId/touch/up"));
+
+    defineCommand(SEND_COMMAND_TO_BROWSER, post("/session/:sessionId/chromium/send_command_and_get_result"));
```

Comment:
This is Chrome specific, and so shouldn't be in the general JsonHttpCommandCodec. The best place is `ChromeDriverCommandExecutor`